tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information.yaml

```diff
@@ -5,9 +5,13 @@ certified_by: null
 dataset_uuid: 633a1d4e-cd40-482f-a5dc-d5901c2181c2
 description: null
 params:
-  adhoc_filters: []
+  adhoc_filters:
```

Comment:
I'm not sure if this is related, but when the time filter is changed both Current Enrollees and Course Information go away, they should be excluded from that filter